api/repository/users.repository.ts

```diff
@@ -0,0 +1,12 @@
+import { prisma } from "../utils/prisma";
+
+export const userRepository = {
+  checkUserExists,
+};
+
+async function checkUserExists(userId: string) {
+  const user = await prisma.user.findUnique({
+    where: { id: userId },
+  });
+  return !!user;
```

Comment:
another way: 
count the number of the users matching the userId ( 0 or 1) and return it

```suggestion
  const user = await prisma.user.count({
    where: { id: userId },
  });
  return user;
```